package.json

```diff
@@ -6,19 +6,50 @@
   "dependencies": {
     "a-sync-waterfall": "^1.0.0",
     "asap": "^2.0.3",
+    "postinstall-build": "^5.0.1",
     "yargs": "^3.32.0"
   },
   "browser": "./browser/nunjucks.js",
   "devDependencies": {
+    "babel-cli": "^6.26.0",
+    "babel-core": "^6.26.0",
+    "babel-loader": "^7.1.2",
+    "babel-plugin-istanbul": "^4.1.5",
+    "babel-plugin-module-resolver": "3.0.0-beta.5",
+    "babel-plugin-transform-runtime": "^6.23.0",
+    "babel-preset-env": "^1.6.1",
+    "babel-register": "^6.26.0",
+    "babel-runtime": "^6.26.0",
+    "connect": "^3.6.5",
+    "cross-env": "^5.1.3",
     "eslint": "^4.13.0",
     "expect.js": "*",
     "express": "4.x",
-    "istanbul": "0.3.x",
+    "get-port": "^3.2.0",
     "mocha": "*",
-    "node-libs-browser": "^0.4.3",
+    "mocha-phantomjs-core": "^2.1.2",
+    "mocha-phantomjs-istanbul": "0.0.2",
+    "node-libs-browser": "^0.7.0",
+    "nyc": "^11.4.1",
+    "phantomjs-prebuilt": "^2.1.16",
+    "serve-static": "^1.13.1",
     "supertest": "*",
-    "uglify-js": "*",
-    "webpack": "^1.8.11"
+    "uglify-js": "^2.8.29",
+    "uglifyjs-webpack-plugin": "^1.1.6",
+    "webpack": "^3.10.0"
+  },
+  "buildDependencies": {
```

Comment:
This is something new. What's it used for?

Reply:
This is a feature of [postinstall-build](https://www.npmjs.com/package/postinstall-build#specifying-build-dependencies-in-packagejson). It allows you to specify a subset of your `devDependencies` necessary to run the `postinstall` hook (when doing a git npm install, for instance).

Reply:
Oh my, I already imagine like `npm install` in Kotsu adding few other nice minutes of longe waiting just while Nunjucks install all those dependencies to build itself...

Reply:
This only happens if the installation is from git (the full command is `postinstall-build src`; the argument `src` is what changes the behavior: if the `src` folder is absent (which, based on your suggestions I will rename dist or lib) then it installs the build dependencies and calls `npm run build`, otherwise it does nothing). Does that address your concerns about build duration?

Reply:
As I've already replied in https://github.com/mozilla/nunjucks/pull/1053#discussion_r162205497, I've got it totally wrong, so my objection above no longer valid :)